### DISCUSSION TITLE
Capture current website url as meta variable

### DISCUSSION THREAD

**Discussion by royfrancis** [quarto-dev/quarto-cli#8024] [Q&A]
### Description

Is it possible to capture the current website url as a meta variable? is it somewhere in the environment?

**Comment by cderv** [comment]
> Is it possible to capture the current website url as a meta variable?

What is the current website url ? Do you mean the current file path viewed ? 

A website url will depend on where you will host your site. Like `github.io` or `netlify`. This happens after the website has been rendered to HTML by Quarto. 

So if you need to tweak some information dynamically based on current url in the browser, you will need JS to run in the browser and do some modification. 

Otherwise, you need to know the website beforehand and pass it as metadata. This is for example required for RSS feed in blog output with `site_url` (https://quarto.org/docs/websites/website-blog.html#rss-feed)

**Reply by royfrancis** [reply]
Sorry! Should've mentioned that this is an OJS question.

**Reply by cderv** [reply]
Not sure to follow... it would be better if you provide an example and more details on what you want to do exactly.
